testing/tests/DevExpress.localization/localization.base.tests.js

```diff
@@ -802,7 +808,6 @@ QUnit.test("format as LDML pattern with custom separators", function(assert) {
 QUnit.module("Localization currency");
 
 QUnit.test("format: base", function(assert) {
-    //assert.equal(currencyLocalization.format(1.2), "$1.2");
```

Comment:
Why is this test case removed?

Reply:
I have just remove commented strings from the code